omero/developers/Model/EveryObject.txt

```diff
@@ -4,14 +4,6 @@ Glossary of all OMERO Model Objects
 Overview
 --------
 
-In navigating the model objects used by the :doc:`OMERO API
```

Comment:
We could certainly make use of `literalinclude` if we want to preserve non-generated code in this documentation repository only